suite-native/graph/src/reducers/types.ts

```diff
@@ -0,0 +1,8 @@
+export type TimeFrameValues = 'hour' | 'day' | 'week' | 'month' | 'year' | 'all';
```

Comment:
why is this file in `reducers` folder?

Reply:
I wanted to have everything related to the logic of package in one folder. I copied `/reducers/packageNameReducer.ts` naming convention from settings, but maybe I could rename the folder to sth else. Any idea? Or I can keep this files in src folder of this pkg.

Reply:
> Or I can keep this files in src folder of this pkg

I would go that way... the file `types.ts` in src.

Reply:
Definitely in root, no reason to keep it only in reducers folder when it's used also in components etc.

Reply:
What's the reason to have reducer in folder then? Because then it's just useless folder for 1 file that can be moved level down. So I think we should either move everything from this folder to src, or group them together here with different name? This would then be useless nesting of folders and creates inconsistencies. What do you think? Move reducer to root as well?

Reply:
Yea, why not keep it flat. We can create folder later if there will be more reducers.

Reply:
This is just a suggestion, what do you think?
https://github.com/trezor/trezor-suite/pull/5838/commits/718a455a86a047e0ce3636bde0cd4e79b3a96fc7